clifford/invariant_decomposition.py

```diff
@@ -0,0 +1,147 @@
+"""
+.. currentmodule:: clifford.invariant_decomposition
+
+=====================================================
+invariant_decomposition (:mod:`clifford.invariant_decomposition`)
+=====================================================
+
+.. versionadded:: 1.4.0
```

Comment:
What should I set in this line, which I just got from copy pasting the Taylor expansion docstring? This is currently causing the docs to fail building.

Reply:
I don't understand why the docs are not building... 
The error I'm seeing in the logs is:
```
jinja2.exceptions.UndefinedError: 'sphinx.config.Config object' has no attribute 'current_version'
```
Perhaps we just are using an old version of sphinx...
@eric-wieser any ideas?

Reply:
You need to merge master, I already fixed this

Reply:
Oops missed the commit, sorry